circle.yml

```diff
@@ -14,7 +14,7 @@ dependencies:
     - ~/pine/target/resolution-cache
 test:
   override:
-    - ls
+    - sbt scalastyle
```

Comment:
Nit: Shouldn't this also call the test target?  In the long run this whole file need to be brought in line with our newer standards.